jobs/ci-kubernetes-node-kubelet-dockerized.sh

```diff
@@ -23,7 +23,7 @@ set -o xtrace
 readonly testinfra="$(dirname "${0}")/.."
 
 export KUBE_VERIFY_GIT_BRANCH='master'
-export KUBE_TEST_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh test/e2e_node/jenkins/jenkins-ci.properties"
+export KUBE_TEST_SCRIPT="${testinfra}/jenkins/node-runner.sh ${testinfra}/jenkins/e2e-image/gcloud-common.sh test/e2e_node/jenkins/e2e-node-jenkins.sh test/e2e_node/jenkins/jenkins-ci.properties"
 
 ### Runner
 readonly runner="${testinfra}/jenkins/gotest-dockerized.sh"
```

Comment:
The advantage of doing this is that we can reuse the logic in `gotest-dockerized.sh` to choose an appropriately-versioned image to use, along with not repeating all of the arguments we supply to `docker`. Though we still need to implement the former for `kubekins-e2e`, so we might want to refactor the logic out somewhere.

Maybe bootstrap should learn how to launch docker directly, rather than requiring an intermediate shell script?